package-lock.json

```diff
@@ -113,7 +113,7 @@
     },
     "harmonizer": {
       "name": "@apollo/harmonizer",
-      "version": "0.2.4",
+      "version": "0.2.5",
       "license": "MIT",
       "dependencies": {
         "@apollo/federation": "file:../federation-js"
```

Comment:
It would be reasonable for you to remove these changes from this PR.  We should land them, but they're not related to it and another PR may make similar conflicting changes.  We should just fix it on `main`.   (Concretely: The best thing for you to do would probably just back-out this file's changes.)